swap/src/protocol/bob.rs

```diff
@@ -1,24 +1,22 @@
 //! Run an XMR/BTC swap in the role of Bob.
 //! Bob holds BTC and wishes receive XMR.
-use self::{amounts::*, message0::*, message1::*, message2::*, message3::*};
-use crate::{
-    network::{
-        peer_tracker::{self, PeerTracker},
-        transport::SwapTransport,
-        TokioExecutor,
-    },
-    SwapAmounts,
-};
+pub use self::{amounts::*, message0::*, message1::*, message2::*, message3::*, state::*};
```

Comment:
So `messageN` exports a `MessageNBehaviour`, `MessageN` and `OutEvent`.

I think it would be cleaner to:
- rename the types to `Message` (or event `Content` or `Body`) and `Behaviour`
- Not do a wildcard re-export (remove `pub use`)
- Use `message0::Behaviour` and `message0::Content` wherever needed.

Reply:
hmm I didn't `messageN::MessageN` to `messageN::Content` because it used in a lot in the rest of the codebase and I wanted to avoid having `alice::messageN::Content` everywhere.